platform/ios/jazzy.yml

```diff
@@ -23,55 +26,62 @@ custom_categories:
       - MGLMapViewDelegate
       - MGLStyle
       - MGLUserTrackingMode
-  - name: Annotations
+  - name: Protocols and Abstract Classes
     children:
       - MGLAnnotation
-      - MGLAnnotationImage
-      - MGLAnnotationView
-      - MGLCalloutView
-      - MGLCalloutViewDelegate
+      - MGLOverlay
+      - MGLShape
       - MGLMultiPoint
-      - MGLMultiPolygon
-      - MGLMultiPolyline
+      - MGLFeature
+  - name: Primitive Shapes
+    children:
       - MGLPointAnnotation
       - MGLPointCollection
       - MGLPolygon
       - MGLPolyline
-      - MGLOverlay
-      - MGLShape
+      - MGLMultiPolygon
+      - MGLMultiPolyline
       - MGLShapeCollection
+  - name: Annotations
+    children:
+      - MGLAnnotationImage
+      - MGLAnnotationView
       - MGLUserLocation
       - MGLUserLocationAnnotationView
-  - name: Map Data
+  - name: User Interaction
+    children:
+      - MGLCalloutView
+      - MGLCalloutViewDelegate
+  - name: Style Primitives
     children:
-      - MGLFeature
-      - MGLMultiPolygonFeature
-      - MGLMultiPolylineFeature
-      - MGLPointCollectionFeature      
       - MGLPointFeature
       - MGLPolygonFeature
       - MGLPolylineFeature
+      - MGLMultiPolygonFeature
+      - MGLMultiPolylineFeature
+      - MGLPointCollectionFeature
       - MGLShapeCollectionFeature
+  - name: Style Content
+    children:
+      - MGLSource
+      - MGLTileSource
+      - MGLShapeSource
+      - MGLRasterSource
+      - MGLVectorSource
   - name: Style Layers
     children:
+      - MGLStyleLayer
+      - MGLForegroundStyleLayer
       - MGLBackgroundStyleLayer
+      - MGLRasterStyleLayer
+      - MGLVectorStyleLayer
       - MGLCircleStyleLayer
       - MGLFillStyleLayer
-      - MGLForegroundStyleLayer
       - MGLLineStyleLayer
-      - MGLRasterStyleLayer
-      - MGLStyleLayer
       - MGLSymbolStyleLayer
-      - MGLVectorStyleLayer
-  - name: Content Sources
+  - name: Style Values
     children:
-      - MGLAttributionInfo
-      - MGLRasterSource
-      - MGLShapeSource
-      - MGLSource
-      - MGLTileCoordinateSystem
-      - MGLTileSource
-      - MGLVectorSource
+      - MGLStyleValue
```

Comment:
What about MGLConstantStyleValue or MGLStyleFunction? MGLStyleValue is a class cluster that includes those two classes, so the developer could get all the information they need with just MGLStyleValue, but it seems odd to have a category with only one item. If the idea is to deemphasize MGLConstantStyleValue and MGLStyleFunction in favor of MGLStyleValue, perhaps we could move this class up to “Maps” alongside MGLStyle.